.travis.yml

```diff
@@ -2,8 +2,6 @@ language: rust
 dist: xenial
 rust:
   - stable
-  - beta
```

Comment:
Why is the CI config modified?

Reply:
I'll remove this commit at the end, it is to just make builds faster.